modules/ROOT/pages/errors/gql-errors/42N0D.adoc

```diff
@@ -0,0 +1,15 @@
+= 42N0D
+
+
+== Status description
+
+error: syntax error or access rule violation - cannot call function from this context. The function `{ <<fun>> }` cannot be called from the current context. It can only be used `{ <<context>> }`.
+
+
```

Comment:
Please add an explanation and example for each of these three new codes
```
== Explanation
<what would cause a user to see this error>

== Example scenario
For example, try to <invalid scenario>:

[source,cypher]
----
<invalid query>
----

The query returns an error with GQLSTATUS <code> and the status description:

[source]
----
<filled in version of the description above>
----

To fix this, <how to solve it>:

[source, cypher]
----
<valid query>
----
```
(something alone these lines, doesn't need to be exactly this pattern)

I know most of the existing errors are very bareboned but that was to get all of them in quickly, as we add or update codes we also fill out their docs pages to be more helpful.

Reply:
Hmm. Adding an explanation and examples certainly makes sense for the 42N0D one, but the other two are internal errors which cannot be caused by users (they will never get further than getting a 42N0D error if they try to call the function). 

I could just add explanation to that effect for 42N0E and 42N0F?

Reply:
Even for 42N0D I can't really give examples whilst the feature (and the one function which would cause this error) is hidden behind the internal feature flag...So gave a basic explanation for now.

Reply:
42N0E and 42N0F are really internal application errors, which makes me question whether we should have gql codes for these at all?

Reply:
> 42N0E and 42N0F are really internal application errors, which makes me question whether we should have gql codes for these at all?

If these are internal, I have the same question. Why do they have GQL codes assigned?